pkg/deployment/values.go

```diff
@@ -0,0 +1,38 @@
+package deployment
+
+// Package level constants
+const (
+
+	// images
+	allInOneValue = "jaegertracing/all-in-one"
+	collector     = "jaeger-collector-image"
+	// jaeger
+	jaeger          = "jaeger"
+	jaegerComponent = "jaeger-component"
+	// version
+	versionLabel = "jaeger-version"
+	versionValue = "1.6"
```

Comment:
The version seems to be redifined for every component. If the operator does not allow to use different versions for different binaries then it would be better to export it to one constant reused for all images.